client/src/pages/Lobby/Lobby.tsx

```diff
@@ -50,20 +50,36 @@ const Lobby: FC<LobbyProps> = (props: LobbyProps) => {
     return game.players && game.players[0].id === playerId
   }
 
-  const defaultSettings: SettingsType = {
-    difficulty: Difficulty.EASY,
-    roundsToWin: 10,
-    cardSpeed: 1.5
-  }
-
   return (
     <div className='Lobby'>
-      {game && (
+      {game && game.players.length && hubConnection && (
         <>
-          <Settings settings={defaultSettings} />
           <h1>Details de la partie ({game.id})</h1>
           <h2>{`Créé ${moment.utc(game.creationDate).fromNow()}`}</h2>
-          <div className='Players'>
+          {isGameHost(game, currentPlayerId) ? (
+            <Settings
+              difficulty={game.difficulty}
+              roundsToWin={game.roundsToWin}
+              cardSpeed={game.cardSpeed}
+              updateSettings={(settings: SettingsType) =>
+                hubConnection.invoke('UpdateGameSettings', settings)
+              }
+            />
+          ) : (
+            <ul>
+              <li>
+                Niveau de difficulté: <b>{game.difficulty}</b>
```

Comment:
Ok boomer.

Reply:
It's best to use a `<strong>` tag instead. `<b>` tag is mostly used for styling purpose and HTML is not about styling things. `<strong>` has a semantic meaning.